src/App.tsx

```diff
@@ -1,61 +1,144 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { ToDo } from './types/type';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(id: number) {
+  return usersFromServer.find(usr => (id) === usr.id) || null;
```

Comment:
brackets around id are redundant here, just fix it to be:

`.find(usr => id === usr.id) || null`